content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md

```diff
@@ -17,6 +17,22 @@ weight: 1
 
 # Corda: Enterprise Edition release notes
 
+## Corda: Enterprise Edition 4.9.3
+
+Corda Enterprise 4.9.3 is a patch release of Corda Enterprise focused on resolving issues.
+
+### Upgrade recommendation
+
+As a developer or node operator, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.9/enterprise.html) as soon as possible. The latest Corda Enterprise release notes are on this page, and you can find the latest upgrade guide [here](../../../../../en/platform/corda/4.9/enterprise/upgrading-index.md).
+
+### Fixed issues
+
+The following issues were resolved in this patch release:
+
+* Previously, both Corda Nodes certificates possessed Certificate Authority power because the CA attribute was set to true. The node registration tool now has a new  option, ```--node-is-ca```, allowing a node legal identity certificate to be created where it is not a CA.
```

Comment:
The option is wrong. I have updated the issue https://r3-cev.atlassian.net/browse/ENT-6859
With the correct details. Should be ....
-C, --node-identity-cert-not-ca

Reply:
Thanks Adel, have just pushed an update there.